### PR TITLE
Homework Databases week#2 Implemented

### PR DESCRIPTION
**Pull request by DiegoRuiz454**

### CODE REVIEW

File: databases/week2/databases_week2.sql [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+
+-- Change a task status
+
+
+UPDATE my_market SET status_id = '9' WHERE idmy_market = 2;
+
+-- Mark a task as complete
+
+
+SELECT * FROM my_market LIMIT 1;
```
Comment (Ekamba) [inline databases/week2/databases_week2.sql]: this sql will not mark a task as complete, instead it will just print one row.

**Review by Ekamba — COMMENTED** [review]
In general you understand the topic you have been learned so far, keep it up 👍

File: databases/week2/databases_week2.sql (hunk 1 above)
Comment (DiegoRuiz454) [reply databases/week2/databases_week2.sql]: thank you so much Ekamba for help

File: databases/week2/databases_week2.sql [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+
+-- Part 1: Working with tasks
+
+
+CREATE SCHEMA `my_work` DEFAULT CHARACTER SET utf8 ;
+
+
+CREATE TABLE `my_work`.`my_market` (
+
+`idmy_market` INT NOT NULL AUTO_INCREMENT,
```
Comment (abedsujan) [inline databases/week2/databases_week2.sql:12]: you can simply call it 'id'

File: databases/week2/databases_week2.sql [hunk 3]
```diff
@@ -0,0 +1,487 @@
...
+
+
+
+-- Part 4: Creating a database
+
+
+
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
```
Comment (abedsujan) [inline databases/week2/databases_week2.sql:343]: do you need these SET statements?

File: databases/week2/databases_week2.sql [hunk 4]
```diff
@@ -0,0 +1,487 @@
...
+
+---------------------------------------------------------
+
+Table `creating_database`.`kitchen`
+
+-------------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS `creating_database`.`kitchen` (
+
+`idkitchen` INT NOT NULL AUTO_INCREMENT,
```
Comment (abedsujan) [inline databases/week2/databases_week2.sql:396]: always try to use a simple column or attribute name. typically people use 'id' as the primary key

File: databases/week2/databases_week2.sql [hunk 5]
```diff
@@ -0,0 +1,487 @@
...
+
+CREATE TABLE IF NOT EXISTS `creating_database`.`kitchen_has_students` (
+ 
+`kitchen_idkitchen` INT NOT NULL,
+
+`students_idstudents` INT NOT NULL,
+ 
+PRIMARY KEY (`kitchen_idkitchen`, `students_idstudents`),
+ 
+INDEX `fk_kitchen_has_students_students1_idx`
```
Comment (abedsujan) [inline databases/week2/databases_week2.sql:427]: I hope you understood the concept of INDEX and `fk_kitchen_has_students_students1_idx`.
What is the use case of VISIBLE?

**Pull request merged**